package.json

```diff
@@ -6,11 +6,15 @@
   "author": "Shahed Nasser",
   "license": "ISC",
   "scripts": {
+    "start": "npm run watch:all",
     "compile": "lessc src/sbuttons.less dist/sbuttons.css && npm run prettier && npm run clean-css",
     "watch": "concurrently \"less-watch-compiler src dist\" \"onchange -v dist/sbuttons.css -- npm run clean-css \" ",
     "prettier": "prettier --write \"src/**/*.less\" \"dist/sbuttons.css\" \"assets/**\" ",
+    "lite": "lite-server",
     "check-prettier": "prettier --check \"src/**/*.less\" \"dist/sbuttons.css\" ",
-    "clean-css": "cleancss -o dist/sbuttons.min.css dist/sbuttons.css"
+    "clean-css": "cleancss -o dist/sbuttons.min.css dist/sbuttons.css",
+    "watch:less": "onchange -v src/*.less -- npm run compile",
```

Comment:
Please remove the "watch" command and rename "watch:less" to just watch. Or rename "watch:all" to "watch". We don't need the previous watch command anymore.